src/components/FullTextView/FullTextView.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import {createDriverFactory} from 'wix-ui-test-utils/driver-factory';
+import {isEnzymeTestkitExists} from 'wix-ui-test-utils/enzyme';
+import {isTestkitExists} from 'wix-ui-test-utils/vanilla';
+import {mount} from 'enzyme';
+import {fullTextViewDriverFactory} from './FullTextView.driver';
+import {FullTextView} from './FullTextView';
+
+import {fullTextViewTestkitFactory} from '../../testkit';
+import {fullTextViewTestkitFactory as enzymeFullTextViewTestkitFactory} from '../../testkit/enzyme';
+
+describe('FullTextView', () => {
+  const createDriver = createDriverFactory(fullTextViewDriverFactory);
+
+  it('should render a span tag by default', () => {
+    const wrapper = createDriver(<FullTextView>Hello World</FullTextView>);
+    expect(wrapper.getTagName()).toBe('span');
+  });
+
+  it('should display full content on hover and hide it on leave in tooltip', async () => {
+    const content = (
+      <div>
+        <span>Delete this super awesome thing</span>
+        <i>?</i>
+      </div>
+    );
+    const component = mount(<FullTextView maxWidth={0}>{content}</FullTextView>);
+
+    expect(component.find('[data-hook="popover-content"]').length).toBe(0);
```

Comment:
can't the Popover testkit be used here instead?

Reply:
yes, but i need to update Popover driver in wix-ui-core

Reply:
you've got it already here
https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/Popover/Popover.driver.ts#L9-L11

Reply:
@shlomitc i know about this) that why i told `update`, current Popover.driver can not solve this problem, let do a short call, i ll explain

Reply:
so let's leave it for later. we can merge it as it is right now